nifi-ngsi-bundle/nifi-ngsi-nar/pom.xml

```diff
@@ -19,11 +19,11 @@
     <parent>
         <groupId>org.apache.nifi</groupId>
         <artifactId>nifi-ngsi-bundle</artifactId>
-        <version>1.3.6</version>
+        <version>2.1.0</version>
```

Comment:
why 2.1.0?

Reply:
it is the latest version in https://github.com/ging/fiware-draco/blob/a4662f421b88e6d68b82c68c73679ad4e29540f4/nifi-ngsi-bundle/nifi-ngsi-nar/pom.xml#L22